ansible_ai_connect_chatbot/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Ansible AI Connect Chatbot"
```

Comment:
We need to parameterize this `Ansible AI Connect` (like the Amin console is doing?), so that downstream can build with the corresponding name

Reply:
I have opened  task [AAP-33903](https://issues.redhat.com/browse/AAP-33903) for these upstream support items.